packages/wrangler/src/versions/upload.ts

```diff
@@ -69,7 +69,6 @@ type Props = {
 	dryRun: boolean | undefined;
 	noBundle: boolean | undefined;
 	keepVars: boolean | undefined;
```

Comment:
Unrelated to this PR, but it looks like these props could do with some cleanup... They seem to take both config.rules and config as separate values?

Reply:
So in the call site for `versionsUpload()` there is `rules: getRules(config)`, which does som additional validation of deprecated rules options. That function is used in various places.

I think that whole function can be removed in Wrangler V4, since the deprecated fields might be removed?